server/core/src/main/scala/sttp/tapir/server/model/EndpointExtensions.scala

```diff
@@ -0,0 +1,33 @@
+package sttp.tapir.server.model
+
+import sttp.tapir.EndpointInfoOps
+import sttp.tapir.AttributeKey
+
+/** Can be used as an endpoint attribute.
+  * @example
+  *   {{{
+  * endpoint.attribute(AttributeKey[MaxContentLength], MaxContentLength(16384L))
+  *   }}}
+  */
+case class MaxContentLength(value: Long) extends AnyVal
+
+object EndpointExtensions {
```

Comment:
we have attributes defined in `Schema`s companion, as well as in `DefaultDecodeFailureHandler` (`OnDecodeFailureNextEndpointAttribute`). Maybe there is a less generic place where this attribute can be specified?

Reply:
`model` seems to be a good package, maybe simply the `MaxContentLength` companion?